test/integration/policy_info_metric_test.go

```diff
@@ -161,7 +160,7 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test policy_governance_info metric
 			}
 
 			return status
-		}, defaultTimeoutSeconds, 1).Should(ContainSubstring("Unauthorized"))
+		}, "90s", 1).Should(ContainSubstring("Unauthorized"))
```

Comment:
(optional) "90s" is probably clearer, but having a multiple follows conventions elsewhere...
```suggestion
		}, defaultTimeoutSeconds*3, 1).Should(ContainSubstring("Unauthorized"))
```